Jamulus.pro

```diff
@@ -94,28 +94,39 @@ win32 {
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
         contains(QT_ARCH, "i386") {
-            message("Native i386 build")
-            libjackname = libjack.lib
+            !exist("C:/Program Files (x86)") {
+                message ("Cross compilation build")
+                programfilesdir = "C:/Program Files (x86)"
```

Comment:
You're in a block that's tested for "not exists" on this directory.  I think you have the logic wrong.

Reply:
Your right. my bad.
7f243127